packages/wallet-standard/src/register.ts

```diff
@@ -0,0 +1,12 @@
+import type { WalletsWindow } from '@wallet-standard/standard';
+import { GlowWallet } from './wallet.js';
+
+declare const window: WalletsWindow;
+
+export function register(): void {
+    try {
+        (window.navigator.wallets ||= []).push(({ register }) => register(new GlowWallet()));
+    } catch (error) {
+        console.error(error);
+    }
+}
```

Comment:
This is the only public export of the package. No errors are ordinarily expected, but in case an extension that is added to the page before Glow does something unexpected to mess with `window.navigator.wallets`, this guard will just log the error instead of causing other issues for dapps.